steps/discover-repos.rb

```diff
@@ -87,7 +87,8 @@
       forks: i[:forks_count],
       created_at: i[:created_at].iso8601,
       size: i[:size],
-      open_issues_count: i[:open_issues_count]
+      open_issues_count: i[:open_issues_count],
+      description: "\"".concat(i[:description], "\"")
```

Comment:
@MoromugHSE I think, just `i[:description]` will be enough, no need to do the contatenation